warehouse/accounts/views.py

```diff
@@ -104,6 +104,8 @@ def login(request, redirect_field_name=REDIRECT_FIELD_NAME,
     # TODO: Logging in should reset request.user
     # TODO: Configure the login view as the default view for not having
     #       permission to view something.
+    if request.authenticated_userid is not None:
+        return HTTPSeeOther(request.route_path('manage.projects'))
```

Comment:
This newly introduced branch doesn't seem to have a test for it.

Reply:
Thanks for pointing out @pradyunsg 

Will address.